Dockerfile

```diff
@@ -1,9 +1,10 @@
 # Docker support, thanks to xinyifly
 
-FROM openjdk:7-alpine
+FROM openjdk:8-alpine
 RUN apk -U add tini
 WORKDIR /mnt
-COPY ./ ./
-RUN sh ./posix-compile.sh
+ADD wz ./wz
+ADD docker-launch.sh configuration.ini world.ini ./
+ADD build/libs/HeavenMS.jar HeavenMS.jar
```

Comment:
This will break the build from a clean clone: `build/libs/HeavenMS.jar: no such file or directory`

Reply:
`./gradlew build` shouldn't be included in the image, as it will increase the image size and never cause containers to cache due to non-deterministic gradle builds and cache files. The project should be built out of docker for this reason.